Content.Server/Chemistry/Components/SolutionManager/RefillableSolutionComponent.cs

```diff
@@ -21,5 +22,21 @@ public class RefillableSolutionComponent : Component
         [DataField("solution")]
         public string Solution { get; set; } = "default";
 
+        /// <summary>
+        /// The maximum amount that can be transferred to the solution at once
+        /// </summary>
+        [DataField("maxrefill")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        public FixedPoint2 MaxRefill { get; set; } = FixedPoint2.New(0);
+
+        /// <summary>
+        /// Does this item have a max refill set?
+        /// </summary>
+        [DataField("cappedfill")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool CappedFill { get; set; } = false;
+
+
+
```

Comment:
Maybe just make MaxRefill nullable & default to null.
```suggestion
        public FixedPoint2? MaxRefill { get; set; } = null;
```

Reply:
I don't have a lot of experience with nullability but I'm assuming it's okay to cast back after I check if it's null? It seems to work when I test it anyway.

Reply:
Yeah. Nullables are a bit funny with structs. You can also just use `MaxRefil.Value` after checking if its null (or alternatively, checking `MaxRefill.HasValue`). But cast also works.